src/main/resources/application.properties

```diff
@@ -4,6 +4,10 @@ spring.datasource.password=${POSTGRES_PASSWORD}
 spring.datasource.driverClassName=org.postgresql.Driver
 spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
 spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.properties.hibernate.generate_statistics=true
+spring.jpa.properties.hibernate.jdbc.batch_size=${BATCH_SIZE}
+spring.jpa.properties.hibernate.order_inserts=true
+spring.jpa.properties.hibernate.order_updates=true
```

Comment:
This file was removed and replaced by `application.yml`. I believe that you didn't update your `main` branch with the one on this repository. 
Please be careful with this. When a PR is merged, you need to `git pull --rebase` your clone.

Reply:
Thanks, I will keep this in mind :+1: